flows/boundary.py

```diff
@@ -568,13 +580,29 @@ async def get_document_passages_from_vespa(
 
         conditions &= text_block_id.contains(text_block_id_contains)
 
+    text_blocks_ids_n: PositiveInt = (
+        VESPA_MAX_LIMIT if text_blocks_ids is None else len(text_blocks_ids)
```

Comment:
thought: Is there a potential issue here where `len(text_block_ids)` is greater than `VESPA_MAX_LIMIT`?

Reply:
There's a check for that [earlier](https://github.com/climatepolicyradar/knowledge-graph/blob/refactor%2Fuse-vespa-connection-pool/flows/boundary.py#L568-L571),